r/aws_cloudfront_distribution_tenant: fix potential hangs by enforcing configured timeouts (#47839)
hashicorp/terraform-provider-aws: website/docs/r/cloudfront_distribution_tenant.html.markdown

## Timeouts

> ~> **NOTE:** In previous versions of this resource, configured timeouts were accepted but not enforced. Timeouts are now fully enforced. If you have previously configured custom timeouts, they will now take effect.

Reviewer: ```suggestion
~> **NOTE:** In versions prior to `v6.51.0`, resource timeouts were accepted but not enforced. Timeouts are now fully enforced. If you have previously configured custom timeouts, they will now take effect.
```

Author: @jar-b

> suggestion to reference specific version v6.51.0

Applied, thanks. Updated in 15531ca5.
